viite-UI/src/view/NodeForm.js

```diff
@@ -409,8 +409,8 @@
     };
 
     var closeNode = function (cancel) {
+      eventbus.off('change:nodeName change:nodeTypeDropdown change:nodeStartDate junction:validate junction:setCustomValidity');
```

Comment:
This list contains now more events than before. I think eventbus.off should be called for all places where there is eventbus.on for example node:selected is still missing

Reply:
The listener for node:selected is only created when `bindEvents()` is called on initialization so there is no need to remove the listener. But for the listeners created inside node:selected it's true, I will add them.

Reply:
...but you are right, the node:selected listener still needs to be removed. I will correct that.